Jenkinsfile

```diff
@@ -23,12 +23,13 @@ clientsTemplate{
         sh "git remote set-url origin git@github.com:fabric8io/fabric8-maven-plugin.git"
 
         def pipeline = load 'release.groovy'
+        
+        // Test for both CI and CD flows till we have confidence in the process.
+        sh "mvn clean install"
```

Comment:
This will work.
It would be better if we can run `mvn verify` or equivalent in CD block.